resources/qml/UserProfile.qml

```diff
@@ -0,0 +1,169 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.3
+import QtQuick.Layouts 1.2
+import QtQuick.Window 2.3
+
+import im.nheko 1.0
+
+import "./device-verification"
+
+ApplicationWindow{
+    property var user_data
+    property var colors: currentActivePalette
+
+    id:userProfileDialog
+    height: 500
+    width: 400
+    modality:Qt.WindowModal
+    Layout.alignment: Qt.AlignHCenter
+    palette: colors
+
+    UserProfileList{
+        id: userProfileList
+        userId: user_data.userId
+        onUserIdChanged : {
+            userProfileList.updateDeviceList()
+        }
+        onDeviceListUpdated : {
+            modelDeviceList.deviceList = userProfileList
+        }
+    }
+
+    Component {
+		id: deviceVerificationDialog
+		DeviceVerification {}
+	}
+    Component{
+        id: deviceVerificationFlow
+        DeviceVerificationFlow {}
+    }
+
+    background: Item{
+        id: userProfileItem
+        width: userProfileDialog.width
+        height: userProfileDialog.height
+        anchors.margins: {
+            top:20
+        }
+
+        ColumnLayout{
+            anchors.fill: userProfileItem
+            width: userProfileDialog.width
+            spacing: 10
+
+            Avatar{
+                id: userProfileAvatar
+                url:chat.model.avatarUrl(user_data.userId).replace("mxc://", "image://MxcImage/")
+                height: 130
+                width: 130
+                displayName: modelData.userName
+                Layout.alignment: Qt.AlignHCenter
+            }
+
+            Label{
+                id: userProfileName
+                text: user_data.userName
+                fontSizeMode: Text.HorizontalFit
+                font.pixelSize: 16
+                color:timelineManager.userColor(modelData.userId, colors.window)
+                font.bold: true
+                Layout.alignment: Qt.AlignHCenter
+            }
+
+            Label{
+                id: matrixUserID
+                text: user_data.userId
+                fontSizeMode: Text.HorizontalFit
+                font.pixelSize: 16
+                color:colors.text
+                Layout.alignment: Qt.AlignHCenter
+            }
```

Comment:
You should probably add the 4 buttons to create a dm, block, ignore and kick someone, or what they were in the old profile back ^-^

Reply:
Totally forgot that

Reply:
I only noticed, since I wanted to dm someone :D